server/__tests__/llm/deepseek.test.ts

```
@@ -0,0 +1,457 @@
+  get: jest.fn(),
+  delete: jest.fn(),
+  head: jest.fn(),
+  options: jest.fn(),
+  put: jest.fn(),
+  patch: jest.fn(),
+  request: jest.fn(),
+  getUri: jest.fn(),
+  postForm: jest.fn(),
+  putForm: jest.fn(),
+  patchForm: jest.fn(),
+} as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)